frontend/src/lib/components/skeletons/SkeletonGraph.tsx

```diff
@@ -0,0 +1,29 @@
+import { Col, Row } from 'antd'
+import clsx from 'clsx'
+import React from 'react'
+import './SkeletonGraph.scss'
+
+interface SkeletonGraphProps {
+    isLoading?: boolean
+}
+
+export function SkeletonGraph({ isLoading }: SkeletonGraphProps): JSX.Element {
```

Comment:
This is great, but the slightly odd thing is that this is a bar chart, so now the skeleton for _all_ insight types is a bar chart. This feels kind of misleading during loading. For non-bar charts I think it may be better to show nothing below – or the relevant viz type of course.

Reply:
Hmmm, I might yank the skeleton graph then. It was fun to build but not fun enough I want to add a tangent of a skeleton for every type 🤣